xdrip-get-entries.sh

```diff
@@ -1232,6 +1244,10 @@ function  post-nightscout-with-backfill()
     # don't post glucose to NS
     #return
   #fi
+  if [ -e "${LDIR}/entry-backfill2.json" ] ; then
+    /usr/local/bin/cgm-post-ns ${LDIR}/entry-backfill2.json && (echo; log "Upload backfill to NightScout worked ... removing ${LDIR}/entry-backfill2.json"; rm -f ${LDIR}/entry-backfill2.json) || (echo; log "Upload backfill to NS did not work ... keeping for upload when network is restored ... Auth to NS may have failed; ensure you are using hashed API_SECRET in ~/.bash_profile";)
```

Comment:
Should we modify the logging text to show "tx backfill" to differentiate it from internet was down backfill scenario?

Reply:
Yes, and also handle the case when there is nothing to upload better.